ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -904,3 +906,70 @@ def test_ipahealthcheck_trust_agent_member_check(self):
         for check in data:
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] == self.master.hostname
+
+
+class TestIpaHealthCheckFilesystemSpace(IntegrationTest):
+    """
+    ipa-healthcheck tool test for running low on disk space.
+    """
+    num_replicas = 1
```

Comment:
Test don't require replica or are you planning to add test on replica as well?

Reply:
Good catch, removed.